src/runtime/virtcontainers/qemu.go

```diff
@@ -685,6 +687,23 @@ func (q *qemu) CreateVM(ctx context.Context, id string, network Network, hypervi
 		qemuConfig.Devices = q.arch.appendPCIeRootPortDevice(qemuConfig.Devices, hypervisorConfig.PCIeRootPort, memSize32bit, memSize64bit)
 	}
 
+	if !qemuConfig.Knobs.Daemonize {
+		qemuConfig.LogFunc = func(pid int, stderr io.ReadCloser) {
+			q.Logger().Infof("Start logging QEMU (qemuPid=%d)", pid)
+			scanner := bufio.NewScanner(stderr)
+			for scanner.Scan() {
+				warnRE := regexp.MustCompile("^[^:]+: warning:")
+				text := scanner.Text()
+				if warnRE.MatchString(text) {
+					q.Logger().WithField("qemuPid", pid).Warning(text)
```

Comment:
Unless I'm mistaken, you are keeping the `warning: ` stanza here. Maybe strip it from the text before passing it to `Warning()`?